src/app/routes/home/getInitialData/index.js

```diff
@@ -14,8 +17,30 @@ const transformJson = pipe(
   filterGroupsWithoutStraplines,
 );
 
-export default async ({ path }) => {
-  const { json, ...rest } = await fetchPageData(path);
+export const hasRadioSchedule = async (service, variant) => {
+  const config = await getConfig(service, variant);
+
+  const serviceHasRadioSchedule = pathOr(
+    false,
+    ['radioSchedule', 'hasRadioSchedule'],
+    config,
+  );
+
+  const radioScheduleOnFrontPage = pathOr(
+    false,
+    ['radioSchedule', 'onFrontPage'],
+    config,
+  );
+
+  return serviceHasRadioSchedule && radioScheduleOnFrontPage;
+};
+
+export default async ({ path, service, variant }) => {
+  const pageHasRadioSchedule = await hasRadioSchedule(service, variant);
+
+  const { json, ...rest } = pageHasRadioSchedule
+    ? await withRadioSchedule(fetchPageData(path), service, path)
```

Comment:
Is it worth changing this to accept an object?

Reply:
Done!